<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissingDumpsViewer</class>
 <widget class="QDialog" name="MissingDumpsViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Missing dumps</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QToolButton" name="toolButtonClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Close this window</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="toolTip">
      <string>List of missing dumps</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Set / ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CRC</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SHA1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reason</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frameExport">
     <layout class="QGridLayout" name="gridLayout2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButtonExportToDataFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Export missing dumps to a data file</string>
        </property>
        <property name="text">
         <string>Export to data file</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBoxSelectedDumpsOnly">
        <property name="toolTip">
         <string>Export only dumps that are selected in the above list (or all)</string>
        </property>
        <property name="text">
         <string>Selected dumps only</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>toolButtonExportToDataFile</tabstop>
  <tabstop>checkBoxSelectedDumpsOnly</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>MissingDumpsViewer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>914</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
